src/nbl/builtin/CMakeLists.txt

```diff
@@ -6,11 +6,18 @@ function(nbl_add_builtin_resources)
   list(APPEND builtin_dependencies_list  ${CMAKE_CURRENT_SOURCE_DIR}/builtin/builtinHeaderGen.py)
   list(APPEND builtin_dependencies_list  ${CMAKE_CURRENT_SOURCE_DIR}/builtin/builtinDataGen.py)
 
+  # Just in case if not found yet (3d parties should've found it already though regardless of platform)
+  find_package(PythonInterp 3 REQUIRED)
```

Comment:
> Deprecated since version 3.12: Use [FindPython3](https://cmake.org/cmake/help/latest/module/FindPython3.html#module:FindPython3), [FindPython2](https://cmake.org/cmake/help/latest/module/FindPython2.html#module:FindPython2) or [FindPython](https://cmake.org/cmake/help/latest/module/FindPython.html#module:FindPython) instead.

from cmake docs

Reply:
just a little wake up call! on latest MSVC (VS 2022, works fine on VS 2019 though) on this weird build, this has to be `PythonInterp 3` to find the python3_EXECUTABLE otherwise it fails. So, you might want to reconsider.